tools/local-test/scripts/test-by-ibcmd.sh

```
@@ -2,44 +1,107 @@
+
+. $LIB_PATH/print.sh
+
+mkdir -p $LOGS_PATH
+mkdir -p $DB_PATH
+mkdir -p $EXPORT_PATH
+mkdir -p $BINARY_PATH
+
+mkdir -p $TMP_PATH
+mkdir -p $RESULT_PATH
+
+rm -rf $TMP_PATH/*
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)